frontend/website/pages/docs/developers/code-reviews.mdx

```diff
@@ -11,7 +11,7 @@ A good pull request:
 
 ## OCaml things
 
-- Are the [style guidelines](../style-guide) being followed?
+- Are the [style guidelines](https://codaprotocol.com/docs/developers/style-guide) being followed?
```

Comment:
problem with hardcoding these is that when you develop locally you'll see the prod version

Reply:
@michellewong793 We can just use `./style-guide` or `/docs/developers/style-guide`

Reply:
will fix!